pkg/threescale/threescale_test.go

```diff
@@ -48,20 +48,6 @@ func TestHandleAuthorization(t *testing.T) {
 			expectStatus: int32(rpc.INVALID_ARGUMENT),
 			expectErrMsg: []string{"error building HTTP client for 3scale system", "invalid URI for request"},
 		},
-		{
-			name: "Test fail - missing request path",
-			params: pb.Params{
-				ServiceId:   "123",
-				SystemUrl:   "https://www.fake-system.3scale.net",
-				AccessToken: "789",
-			},
-			expectStatus: int32(rpc.INVALID_ARGUMENT),
-			expectErrMsg: []string{"missing request path"},
-			template: authorization.InstanceMsg{
-				Name:   "",
-				Action: &authorization.ActionMsg{},
-			},
-		},
```

Comment:
I'm not convinced about the reason to remove this test. If we require the path (and we do), then a test ensuring we error out correctly if it is missing is useful because the possibility for that to happen exists, even if it is a human/integration error.

Reply:
Well, currently the only way to hit this handler is through an `instance` CR where we default an empty path to `/`. 

If for some reason the user messed up the config, wouldn't we not want the same behaviour in the code?

Reply:
Its not a part of the config that the user should ever modify btw

Reply:
You can take the logic from the instance CR and say: "why would we ever have an empty path, and if we had one, what use would it be to run the request through us?".

We really are next to useless with no real path information, so making it very evident we are being useless is important in my opinion. So assuming we should never have a really empty path if the instance CR was correct, the next way for us to have an empty request path would be that the user messed it up. Again, the effect is we become next to useless, so I'd say make it painfully clear we are not adding value by breaking early.

So in the instance CR, we'd not set the path if there is nothing meaningful to set it to, and then in the code we'd barf if we saw an empty path, which would mean: istio logic was nuked and some huge bug happened (or some broken networking, or...), or the user misconfigured it, and in either case, we really have little to do with that information, and most likely it would be erroneous either way, since `/` is not the same as `/rob?what=thebank`, for example.

Reply:
Fixed